graphite/graphite.go

```diff
@@ -2,6 +2,7 @@
 package graphite // import "bosun.org/graphite"
 
 import (
+	"bosun.org/cache"
```

Comment:
This should not be grouped with the stdlib imports. Move it down with a newline separating